pylint init_screen.py

************* Module init_screen
src/init_screen.py:1:0: C0114: Missing module docstring (missing-module-docstring)
src/init_screen.py:8:0: C0116: Missing function or method docstring (missing-function-docstring)
src/init_screen.py:29:29: E1101: Module 'pygame' has no 'QUIT' member (no-member)

--- a/src/init_screen.py
+++ b/src/init_screen.py
@@ -1,41 +1,38 @@
-# Importa e inicia pacotes
-from os import path
+"""Módulo para inicializar a tela de jogo com pygame e gerenciar a interface do usuário."""
+
 import pygame
 from config import BLACK, WIDTH, HEIGHT, GAME, QUIT
 from assets import BACKGROUND, BOTAO_JOGAR, load_assets
 from sprites import Button
 
 def init_screen(window):
+    """Inicializa a tela do jogo carregando recursos e preparando o ambiente gráfico."""
     assets = load_assets()
 
-    # Carrega o fundo da tela inicial
     background = assets[BACKGROUND]
     background_rect = background.get_rect()
 
     img_botao = assets[BOTAO_JOGAR]
-    botao = Button(WIDTH/2-224, HEIGHT/2+15, img_botao, 0.5)
+    botao = Button(WIDTH // 2 - 224, HEIGHT // 2 + 15, img_botao, 0.5)
 
     pygame.mixer.music.play(loops=-1)
     running = True
+    state = None
     while running:
-
-         # A cada loop, redesenha o fundo e os sprites
         window.fill(BLACK)
         window.blit(background, background_rect)
         botao.draw(window)
-        # Processa os eventos (mouse, teclado, botão, etc).
+
         for event in pygame.event.get():
-            # Verifica se foi fechado.
             if event.type == pygame.QUIT:
                 state = QUIT
                 running = False
 
-            if botao.press():
-                state = GAME
-                running = False
+            elif event.type == pygame.MOUSEBUTTONDOWN:
+                if botao.press():  # Ajustado conforme a definição do método ‘press’
+                    state = GAME
+                    running = False
 
-        # Depois de desenhar tudo, inverte o display.
         pygame.display.flip()
 
-        pygame.display.update()
     return state

